stat/distuv/statdist.go

```diff
@@ -10,6 +10,69 @@ import (
 	"gonum.org/v1/gonum/mathext"
 )
 
+// Bhattacharyya is a type for computing the Bhattacharyya distance between
+// probability distributions.
+//
+// The Bhattacharyya distance is defined as
+//  D_B = -ln(BC(l,r))
+//  BC = \int_-∞^∞ (p(x)q(x))^(1/2) dx
```

Comment:
`\int_-∞^∞` looks a bit funny.
I am not strongly opinionated on this issue but perhaps we could just use a latex-ish syntax here as well?

Reply:
It is latex-ish. It could be made real latex with `\int_{-\infty}^\infty` but that seems like the wrong balance in clarity.